docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,355 @@
+
+Also, a CDH cluster’s `HADOOP_CONF_DIR` should be `/etc/hadoop/conf` on CDH by default.
+
+# 3. Run Jobs on Yarn
+In the following part, we will show you three ways that BigDL supports to submit and run our programs on Yarn:
+* Use a built-in function
+* Use a BigDL provided script
+* Use a spark-submit script
+
+## 3.1 Run Programs with built-in function
+This is the easiest and most recommended way to run BigDL on YARN, you only need to prepare the environment on the driver machine, all dependencies would be automatically packaged and distributed to the whole Yarn cluster. In this way, you can easily switch your job between local (for test) and YARN (for production) by changing the `cluster_mode` to be yarn-client or yarn-cluster, `init_orca_context` will submit the job to YARN with client or cluster mode respectively.
+
+### 3.1.1 Yarn Client
+When running every Orca program on yarn-client mode using built-in function, we should call `init_orca_context` to create an OrcaContext at the very beginning of our programs.
+
+```python
+from bigdl.orca import init_orca_context
+
+init_orca_context(cluster_mode="yarn-client", cores=4, memory="10g", num_nodes=2)
+```
+
+To execute the tutorial provided Fashion-MINIST example, we should set `--cluster_mode` argument as `yarn-client` or `yarn` to create an OrcaContext and run the program on yarn-client mode as below:
+
+```bash
+python train.py --cluster_mode yarn-client
+```
+
+### 3.1.2 Yarn Cluster
+Fisrt, please call `init_orca_context` to create an OrcaContext at the very beginning of Orca programs. To run programs on yarn cluster mode using built-in function, we should switch the `cluster_mode` to `yarn-cluster` in `init_orca_context` as below:
+
+```python
+from bigdl.orca import init_orca_context
+
+init_orca_context(cluster_mode="yarn-cluster", cores=4, memory="10g", num_nodes=2)
+```
+
+When running the program provided by tutorial, we should:
+* Set `--cluster_mode` parameter to `yarn-cluster`, which could create an OrcaContext and run the Fasion-MNIST program on yarn-cluster mode.
+* Set `--remote_dir` parameter to the datasets path on remote resources. We recommand you to use datasets from remote resources like HDFS and S3 to instead of downloading these datasets, since there may get connection errors on `yarn-cluster` mode.
+
+Now, please run the example as the following script:
+```bash
+python train.py --cluster_mode yarn-cluster --remote_dir hdfs://path/to/remote/data
+```
+
+## 3.2 Run Programs with BigDL Provided Scripts
+For `spark-submit` users, BigDL provides a `bigdl-submit` script, you can find it in `${BIGDL_HOME}/scripts/bigdl-submit` and adjust the configurations according to your cluster settings. 
+
+```bash
+# bigdl-submit script
+spark-submit \
```

Comment:
why `spark-submit`?

Reply:
`bigdl-submit` is renamed from `spark-submit-with-bigdl`, so there is the `spark-submit` in the `bigdl-submit`.

Reply:
you should show how to use `bigdl-submit` command line, instead of showing how it is implemented